test-case/volume-basic-test.sh

```diff
@@ -58,7 +58,7 @@ sleep 1
 [[ ! $(pidof aplay) ]] && die "aplay process is terminated too early"
 
 sofcard=${SOFCARD:-0}
-pgalist=($(amixer -c"$sofcard" controls | grep -i PGA | sed 's/ /_/g;' | awk -Fname= '{print $2}'))
+mapfile -t pgalist < <(amixer -c"$sofcard" controls | grep -i PGA | sed 's/ /_/g;' | awk -Fname= '{print $2}')
```

Comment:
```suggestion
readarray -t pgalist < <(amixer -c"$sofcard" controls | awk -Fname= '/PGA/i {print $2}')
```

Reply:
I've been trying to figure out why the new line fails.  No luck so far.

```
error: log_entry_address          0 is not in dictionary range!
...
2023-06-21 18:30:40 UTC [COMMAND] amixer -c0 cset name=''EQIIR20.0 eqiir coef 20'' 0%
amixer: Cannot find the given element from control sysdefault:0

2023-06-21 18:30:40 UTC [ERROR] amixer return error, test failed
2023-06-21 18:30:40 UTC [ERROR] Starting func_exit_handler(), exit status=1, FUNCNAME stack:
/home/ubuntu/sof-test-greg/case-lib/hijack.sh: line 17: 15280 Killed                  aplay -D "$dev" -c "$channel" -r "$rate" -f "$fmt" /dev/zero
2023-06-21 18:30:40 UTC [ERROR]  func_error_exit()  @  ./test-case/volume-basic-test.sh
2023-06-21 18:30:40 UTC [ERROR]  main()  @  ./test-case/volume-basic-test.sh:79
```

Reply:
This works:

`readarray -t pgalist < <(amixer -c"$sofcard" controls | grep -i PGA | sed 's/ /_/g;' | awk -Fname= '{print $2}')`

Reply:
`declare -p pgalist` shows any difference, add it in the code and you will see what's wrong.

Reply:
> `amixer -c0 cset name=''EQIIR20.0 eqiir coef 20''`

Double quoting?